Quotient/converters.h

```diff
@@ -450,8 +450,8 @@ namespace _impl {
         o.insert(k, toJson(std::forward<ValT>(v)));
     }
 
-    template <typename ValT>
-    inline void addTo(QUrlQuery& q, const QString& k, ValT&& v)
+    inline void addTo(QUrlQuery& q, const QString& k, auto v)
+        requires requires { QStringLiteral("%1").arg(v); }
```

Comment:
i'm slightly afraid to ask, but is `requires requires` really a thing?

Reply:
Yes... Makes me roll my eyes too. TL;DR, the first `requires` defines a _clause_, in which we specify concepts to satisfy, or actually any (to some extent) expression with a boolean value; and the second `requires` makes for that _expression_, together with a certain syntactic construct called to describe which things on the template parameters should exist or otherwise be valid. C++ standard authors treat keywords as an extremely scarce resource, and they went too far in reusing keywords here, in my opinion. See https://stackoverflow.com/questions/54200988/why-do-we-require-requires-requires for details.